src/Microsoft.VisualStudio.ProjectSystem.CSharp/ProjectSystem/LanguageServices/CSharpCommandLineParserProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    internal class CSharpCommandLineParserProvider
+    {
+        [ImportingConstructor]
```

Comment:
We could, but I do for it for a couple of reasons; as soon as you add a constructor import, you need [ImportingConstructor]. It's a common mistake to forget, hence why i like to be very explict.